src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,16 @@
 
 public class MainApp {
 
+    public static void main(String[] args) {
+        Truck truck = new Truck();
```

Comment:
please use an abstract variable type
```suggestion
        Machine truck = new Truck();
```